eng/pipelines/templates/jobs/vmr-synchronization.yml

```diff
@@ -42,25 +42,14 @@ jobs:
         value: https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-dotnet
 
   pool:
-    ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      name: $(DncEngPublicBuildPool)
-      demands: ImageOverride -equals Build.Ubuntu.2004.Amd64.Open
-    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-      name: $(DncEngInternalBuildPool)
-      demands: ImageOverride -equals Build.Ubuntu.2004.Amd64
+    name: $(DncEngInternalBuildPool)
```

Comment:
this looks odd. This template is still referenced by the PR build template

Reply:
Ah nice catch